### PR TITLE
dev-ruby/neovim-ruby-client: version bump to 0.7.0

### PR DESCRIPTION
**Pull request by lluixhi**
Adds multi_json as a dependency

### CODE REVIEW

File: dev-ruby/neovim-ruby-client/neovim-ruby-client-0.7.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+USE_RUBY="ruby22 ruby23 ruby24"
```
Comment (Tranquility) [inline dev-ruby/neovim-ruby-client/neovim-ruby-client-0.7.0.ebuild]: support for 2.2 got removed

Comment (Tranquility) [reply dev-ruby/neovim-ruby-client/neovim-ruby-client-0.7.0.ebuild]: OK, looks like I misread the changelog. The gemspec says >= 2.2

**Comment by graaff** [comment]
This ebuild fails with USE=doc. Easiest remedie is to add RUBY_FAKEGEM_RECIPE_DOC="rdoc".

This ebuild fails with FEATURES=test.

This ebuild still uses EAPI=5. Please consider updating to 6.

**Comment by flappyports** [comment]
no activity

**Pull request closed without merging**